src/basics.php

```diff
@@ -51,7 +51,10 @@ function debug($var, $showHtml = null, $showFrom = true)
         $line = '';
         $lineInfo = '';
         if ($showFrom) {
-            $trace = Debugger::trace(['start' => 1, 'depth' => 2, 'format' => 'array']);
+            $trace = Debugger::trace(['start' => 1, 'depth' => 3, 'format' => 'array']);
+            if (count($trace) > 1) {
```

Comment:
You are right, I had tested only but putting debug() and dd() in controller and got the expected result.  But for e.g. putting debug() in a model class yields incorrect result. I'll if there's a way to properly get the stack frame needed.